eos-tech-support/eos-hack-reset-clubhouse

```diff
@@ -18,4 +18,8 @@ rm -f ~/.var/app/com.endlessm.Fizzics/data/webkitgtk/localstorage/*
 echo "Resetting desktop icons"
 eos-reset-desktop
 
+echo "Resetting cursor"
+dconf reset /org/gnome/desktop/interface/cursor-size
+dconf reset /org/gnome/desktop/interface/cursor-theme
```

Comment:
I think it's more customary to use `gsettings reset` here instead of `dconf` directly, since that works with any gsettings backend, not just the dconf one. Any reason not to use that here?

Reply:
No reason at all, it didn't come to my mind.